Nuevos commits

lautaronmartinez opened the pull request:
- Se agregó el Dockerfile.
- Se comprobó el funcionamiento del servidor FTP.
- Se agregó documentación.
- Se creó el descargador.

lautaronmartinez committed:
Primer commit

lautaronmartinez committed:
Segundo commit

Se agregó el dockerfile y documentación.

lautaronmartinez committed:
Tercer commit

Se separo el Dockerfile del modulo generador y usuario para generar uno independiente para cada uno.

lautaronmartinez committed:
Update README.md

HernanG234 commented on estudiantes/lautaro_martinez/tp/Dockerfile.Generador:
```diff
@@ -0,0 +1,15 @@
+FROM debian:latest
+
+RUN apt-get update -y
+RUN apt-get install bash -y
```
Bash no viene en la imagen por defecto?

HernanG234 commented on estudiantes/lautaro_martinez/tp/Dockerfile.Tp:
```diff
@@ -0,0 +1,22 @@
+RUN apt-get update -y
+RUN apt-get install bash -y
+RUN apt-get install curl -y
+RUN apt-get install cron -y
```
El usuario no necesita cron.

HernanG234 commented on estudiantes/lautaro_martinez/tp/README.md:
```diff
@@ -0,0 +1,30 @@
+# Documentacion TP
```
Gracias por poner el README :pray:

HernanG234 commented on estudiantes/lautaro_martinez/tp/modulo_usuario/menu.sh:
```diff
@@ -0,0 +1,41 @@
+            ./descargador.sh
+
+            # Ejecutar el script de Clasificador de Archivos (clasificador.sh)
+            ./clasificador.sh
```
El clasificador recibe argumentos posicionales de entrada, pero acá no se le pasa ninguno. Anda?

HernanG234 reviewed (changes requested):
Porfa, sacar la carpeta outputs/! :pray: que se me tilda github

lautaronmartinez committed:
Eliminacion de archivos

Se elimino una gran porción de los archivos generados en el test que provocaban que la carpeta outputs sea pesada

lautaronmartinez committed:
Se corrigió el clasificador

HernanG234 commented on estudiantes/lautaro_martinez/tp/outputs.tar:
Podríamos remover todos los archivos generados? Borrarlos y quizás agregarlos a un .gitignore?

HernanG234 reviewed (changes requested):
Está joyita! Solo revisar el último comentario

lautaronmartinez committed:
Se soluciono el problema con el clasificador. Ya todo funciona de manera optima

lautaronmartinez committed:
Commit 4

lautaronmartinez committed:
Converti los docker runs en scripts. Y corregi algunos detalles

lautaronmartinez committed:
me equivoque en la ruta. La corregi

lautaronmartinez committed:
.

HernanG234 reviewed (changes requested):
Podríamos remover todos los archivos generados? Borrarlos y quizás agregarlos a un .gitignore?
No hace falta tener todos los outputs, hay casi 200 archivos en esta PR D:

lautaronmartinez committed:
Listo profe. Ya ignore los archivos.
